src/main/resources/application.properties

```diff
@@ -120,6 +120,7 @@ info.app.version=local-dev
 info.toggles.subscriptionApproval=true
 # Feature toggle for deleting a json schema even if the topic has subscribers
 info.toggles.schemaDeleteWithSub=false
+info.galapagos.instanceName=Confluent Kafka
```

Comment:
Default value should be empty, and a comment should describe what this value does.